conn_test.go

```diff
@@ -1550,6 +1550,27 @@ func TestQuoteIdentifier(t *testing.T) {
 	}
 }
 
+func TestQuoteLiteral(t *testing.T) {
```

Comment:
I would like this test to just be a slice of strings. The test calls QuoteLiteral, puts that string in a select statement, and verifies that the return value is the same is the original string. The current test doesn't test that postgres itself parses the strings correctly. Add entries with non-ascii runes too. Like `\x00` and others.